File: Python/beginner/2006.py
# https://judge.beecrowd.com/pt/problems/view/2006
# Identificando o chá

teaType = int(input())

guesses_str = input().split()

rightGuesses = 0
for guess_str in guesses_str:
    guess = int(guess_str)
    if guess == teaType:
        rightGuesses += 1

print(rightGuesses)

# By: Fernando Serra
# https://github.com/fernandosserra

File: Haskell/beginner/2006.hs
-- https://judge.beecrowd.com/pt/problems/view/2006
-- Identificando o Chá

-- Imports
import Control.Monad (replicateM)

main = do
    teaType <- readLn :: IO Int
    guessesStr <- getLine
    let guesses = map (read :: String -> Int) (words guessesStr)
    let rightGuesses = length $ filter (== teaType) guesses
    print rightGuesses

-- By: Fernando Serra
-- https://github.com/fernandosserra